cmd/populator/cmd/register/ocf_manifests.go

```diff
@@ -26,60 +25,59 @@ import (
 	"go.uber.org/zap"
 )
 
+type sourceInfo struct {
+	dir     string
+	files   []string
+	gitHash []byte
+}
+
 // NewOCFManifests returns a cobra.Command for populating manifests into a Neo4j database.
 // TODO: support configuration both via flags and environment variables
 func NewOCFManifests(cliName string) *cobra.Command {
-	return &cobra.Command{
+	var sources []string
+	cmd := &cobra.Command{
 		Use:   "ocf-manifests [MANIFEST_PATH]",
 		Short: "Populates locally available manifests into Neo4j database",
 		Example: heredoc.WithCLIName(`
 			APP_JSON_PUBLISH_ADDR=http://{HOST_IP} <cli> .
 		`, cliName),
 		Args: cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runDBPopulate(cmd.Context(), args[0])
+			return runDBPopulateWithSources(cmd.Context(), sources)
 		},
 	}
+	flags := cmd.Flags()
+	flags.StringSliceVar(&sources, "source", []string{}, "Manifests location")
+	return cmd
 }
 
-func runDBPopulate(ctx context.Context, src string) (err error) {
+func runDBPopulateWithSources(ctx context.Context, sources []string) (err error) {
 	var cfg dbpopulator.Config
 	err = envconfig.InitWithPrefix(&cfg, "APP")
 	if err != nil {
 		return errors.Wrap(err, "while loading configuration")
 	}
 
-	// setup logger
 	log, err := logger.New(cfg.Logger)
 	if err != nil {
 		return errors.Wrap(err, "while creating zap logger")
 	}
 
-	parent, err := ioutil.TempDir("/tmp", "*-hub-parent")
-	if err != nil {
-		return errors.Wrap(err, "while creating temporary directory")
-	}
-	dstDir := path.Join(parent, "hub")
-	defer func() {
-		if rErr := os.RemoveAll(parent); rErr != nil {
-			err = multierror.Append(err, rErr)
-		}
-	}()
-
-	err = getter.Download(ctx, src, dstDir)
+	sourcesInfo, err := getSourcesInfo(ctx, cfg, log, sources)
```

Comment:
please ignore if a given source is passed multiple times:
```
populator register ocf-manifests --source github.com/capactio/hub-manifests --source github.com/capactio/hub-manifests
```

Reply:
fixed, duplication removed